tests/routines_test.py

```diff
@@ -135,6 +144,7 @@ def test_send_requested_currency(self):
 class BalanceTest(unittest.TestCase):
     """Test getting balance for ETH/BTC addresses"""
 
+# pylint: disable=no-self-use
```

Comment:
@yaroslavr-itd , I think you should add `self.assertSomething` on the return values of the two valid tests. It will get rid of the no-self-use and it actually makes sense to check that you are getting a reasonable response.

Reply:
Fixed in commit fbd72c3c6bd6e4ac18fc831990329864c64d102c